pages/index.vue

```diff
@@ -8,7 +8,16 @@
         v-bind:key="message.ts"
         @showThread="showThread($event)"
       ></message-card>
-
+      <v-form ref="form">
+        <v-textarea
+          v-model="sendingMessage"
+          :counter="max"
+          :rules="rules"
+          label="投稿メッセージ"
+          placeholder="投稿文書、URL"
+        ></v-textarea>
```

Comment:
.vueファイルのtemplateの中では、中身のないコンポーネントはself closing推奨です。
```suggestion
        <v-textarea
          v-model="sendingMessage"
          :counter="max"
          :rules="rules"
          label="投稿メッセージ"
          placeholder="投稿文書、URL"
        />
```

Reply:
勉強になる 👍🏻

Reply:
修正しました。ありがとうございます。
https://github.com/takuuuu517/nuxt-sample-app/pull/3/commits/a89af187e6f9406411f35400f082ce55bd512294